src/main/java/Listener.java

```diff
@@ -18,8 +18,26 @@ private void conlin(MessageReceivedEvent event) {
             "> The limit of the function never depends on the value of the function at that point.\n\n- John Conlin"
         );
     }
-    private void german(MessageReceivedEvent event){
-        sendMessage(event, "Guten Morgen");
+    private void foreign(MessageReceivedEvent event){
+        int check = (int)(Math.Random()*(4-1+1)+1);
```

Comment:
(4-1+1) = 4 so why on earth would u do 4-1+1 its the same thing as if i were to do int four = 4 + 1 -1; please get help i have an immense disdain for this code. i hate it so much why on earth would you write this. this single line of code hurts me more than all of conlin.